ocaml/ocaml-menhir/Portfile

```diff
@@ -4,22 +4,21 @@ PortSystem          1.0
 PortGroup           ocaml 1.1
 
 name                ocaml-menhir
-version             20210310
+version             20230608
 revision            0
 categories          ocaml devel
-platforms           darwin
-maintainers         nomaintainer
-license             {QPL LGPL}
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+license             GPL-2
```

Comment:
Menhir is LGPL-2. It is not GPL-2.

Reply:
Actually I'm mistaken. Part of it is LGPL (the runtime), part is GPL-2 (the generator.)

Reply:
https://gitlab.inria.fr/fpottier/menhir/-/blob/master/LICENSE

Reply:
Thank you, I will fix this in half an hour.

Reply:
@pmetzger Fixed, rebased.